core-management/src/main/java/org/wildfly/extension/core/management/CoreManagementRootResourceDefinition.java

```diff
@@ -47,4 +48,12 @@
     public Collection<AttributeDefinition> getAttributes() {
         return Collections.emptyList();
     }
+
+    @Override
+    public void registerChildren(ManagementResourceRegistration resourceRegistration) {
+        super.registerChildren(resourceRegistration);
+        // Configuration Changes
+        resourceRegistration.registerSubModel(ConfigurationChangeResourceDefinition.INSTANCE);
+    }
```

Comment:
The way child resource registration is done in PersistentResourceDefinition subclasses isn't by overriding registerChildren, it's by overriding getChildren().